config/examples/stm32c0.config

```diff
@@ -1,8 +1,9 @@
 ARCH?=ARM
 TARGET?=stm32c0
-#Supports ED25519/SHA384 or RSA2048/SHA256
-SIGN?=RSA2048
-HASH?=SHA256
+#Supports ED25519/SHA384
+# No longer supported: RSA2048/SHA256
```

Comment:
What happened here? Is this due to the code size you mentioned? Shall we just adjust the default flash layout a bit?

Reply:
Yes the RSA version now overflows the 10KB partition. Ed25519 still fits. We do have a LMS configuration file also, but it carries a 2KB header for the application, so it does not improve much the application size. I'm adding multiple configuration files for this target with different flash geometries